packages/jest-cli/src/reporters/summary_reporter.js

```diff
@@ -240,12 +240,20 @@ class SummaryReporter extends BaseReporter {
   }
 
   _getTestSummary(contexts: Set<Context>, globalConfig: GlobalConfig) {
+    const getMatchingTestsInfo = () => {
+      const prefix = globalConfig.findRelatedTests
+        ? ' related to files matching '
+        : ' matching ';
+
+      return (
+        prefix +
+        testPathPatternToRegExp(globalConfig.testPathPattern).toString()
+      );
+    };
+
     const testInfo = globalConfig.onlyChanged
       ? chalk.dim(' related to changed files')
-      : globalConfig.testPathPattern
-        ? chalk.dim(' matching ') +
-          testPathPatternToRegExp(globalConfig.testPathPattern).toString()
-        : '';
+      : globalConfig.testPathPattern ? getMatchingTestsInfo() : '';
```

Comment:
@thymikee previousaly only 'matching' word was dimmed and not the path pattern. I will keep it same way, is that correct ?

Old ref:
chalk.dim(' matching ') +
          testPathPatternToRegExp(globalConfig.testPathPattern).toString()